src/gui/Preferences.h

```diff
@@ -216,6 +216,14 @@ private slots:
   QSettings::SettingsMap defaultmap;
   QHash<const QAction *, QWidget *> prefPages;
 
-  static Preferences *instance;
-  static const char *featurePropertyName;
+  const char* featurePropertyName=nullptr;
+};
+
+class GlobalPreferences
+{
+public:
+    static Preferences* inst();
+private:
+    static Preferences *instance;
+    static const char *featurePropertyName;
```

Comment:
Actually, this is a constant which is only used internally in Preferences.cc, so it doesn't need to be a class member at all, it can just be a static constant on the global file level (or in an anonymous namespace)

Reply:
Agreed

Reply:
Yes, I meant that it can be removed also from Preferences, as it's a compile-time constant -> it can just be a global constant on the file scope of Preferences.cc

Reply:
Actually I was not sure if this was something that could change for different "Preferences". 
I will remove it.

Reply:
I think it's just some old code which can be cleaned up :)

Reply:
Done.